src/containers/catalog/CatalogEpic.js

```diff
@@ -1,7 +1,9 @@
+/* @flow */
 import * as actionTypes from './CatalogActionTypes';
 import { Permission } from '../../core/permissions/Permission';
 import * as actionFactories from './CatalogActionFactories';
 import { Observable } from 'rxjs';
+import { combineEpics } from 'redux-observable';
 
 const EXAMPLE_ENTITY_SET = {
```

Comment:
is this `EXAMPLE_ENTITY_SET` meant to be shown to the user, or is it just for local development?